specs/credentialmanagement/index.src.html

```diff
@@ -998,8 +1032,8 @@ <h4 id="request-credential">
   {{LocallyStoredCredential}} isn't the only type? For example, how could an
   author pull some "RemoteCredential" that required interaction with a
   third-party, but fall back to a {{PasswordCredential}}? For the moment, we're
-  resolving this by rejecting the promise if the types listed in
-  {{CredentialRequest/types}} aren't all instances of the same type, but that
+  resolving this by rejecting the promise if the types listed as keys in
+  {{CredentialRequestFilters}} map aren't all instances of the same type, but that
```

Comment:
You dropped this constraint above. If you intend to drop it, we'll need to drop this issue as well. As I noted, it's not clear that we can.